lib/sentry_logging.rb

```diff
@@ -65,10 +65,15 @@ def non_nil_hash?(h)
   private
 
   def client_error?(va_exception_errors)
-    va_exception_errors.present? && va_exception_errors.detect { |h| client_error_status?(h[:status]) }.present?
+    va_exception_errors.present? &&
+      va_exception_errors.detect { |h| client_error_status?(h[:status]) || evss_503?(h[:code], h[:status]) }.present?
   end
 
   def client_error_status?(status)
     (400..499).cover?(status.to_i)
   end
+
+  def evss_503?(code, status)
```

Comment:
I think this might be better located in the `EVSS::ErrorMiddleware` class, perhaps on [line 37](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/evss/error_middleware.rb#L37). Reason being, this change is specific to the EVSS service, and we already have a conditional branch in the linked class handling 503 responses, at which point we could change the error level.